scripts/covid19indiaORG/Config.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Covid 19 India has some other data that we don't need for the import.
+# Only store data for the following labels.
+DATA_TO_KEEP = set(['confirmed', 'deceased', 'tested', 'active', 'recovered'])
+
+# Every state has its own API.
+STATE_APIS = {
```

Comment:
could this just be a function from, say, "WB" to "https://api.covid19india.org/v4/min/timeseries-WB.min.json" if the format of the string is the same

Reply:
Done!